zerver/views/registration.py

```diff
@@ -742,7 +752,7 @@ def accounts_home(
                 logging.error("Error in accounts_home")
                 return HttpResponseRedirect("/config-error/smtp")
 
-            return HttpResponseRedirect(reverse("signup_send_confirm", kwargs={"email": email}))
+            return HttpResponseRedirect(reverse("signup_send_confirm") + "?email=" + email)
```

Comment:
We are avoiding constructing URLs with `+` these days. This should be using `append_url_query_string`.

(In particular, I'm very suspicious that doing `+` could do something wrong with encoding)

Reply:
Oh! done.